BDNS_Abbreviations_Register.csv

```diff
@@ -446,6 +446,7 @@ sensor - velocity sensor,VS,SENSOR,IfcSensor,NOTDEFINED
 shading - shading / blinds / drapes device actuator,SDACT,HVAC/SDC,IfcActuator,ELECTRICACTUATOR
 shading - shading / blinds / drapes device keypad,SDKP,,IfcSwitchingDevice,KEYPAD
 signal - beacon,BCN,,ifcAlarm,LIGHT
+switch-differential pressure switch,DPS,,IfcSwitchingDevice,NOTDEFINED
```

Comment:
Please rename to DPRSW so that there is no clash with the existing abbreviations

Reply:
Please update as follows:


actuator - differential pressure switch, DPRSW,,IfcActuator,NOTDEFINED

Reply:
Updated the changes. Thank You